src/schema.graphql

```diff
@@ -0,0 +1,13 @@
+# import Division from "./generated/prisma.graphql"
+
+type Query {
+    info: String!,
+    divisions: [Division!]!
+    division(id: ID!): Division
+}
+
+type Mutation {
+    createDivision(name: String!, description: String): Division!
```

Comment:
try to use input type.Something like this
`input createDivisionInput {
name: String!, description: String
}`

Reply:
What are the benefits of this approach? @vagabondhoang

Reply:
If some things has same input type, u just need to write only once.And it will be better for writing unit test later